### PR TITLE
POR smart contract implementation

### PR DESCRIPTION
**Pull request by bayram98**
# Description

This PR is ready to review and merge.

Changes:
- add script to deploy POR contract
- default deployment configuration
- migration files for `cypress`, `baobab`, and `localhost`
- command to deploy POR contract
 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: contracts/migration/localhost/POR/20230831181923_deploy.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "deploy": {
+    "name": "e-Gold",
+    "timeout": 0,
+    "validator": "0x0000000000000000000000000000000000000000",
+    "decimals": 8,
+    "description": "POR Contract",
+    "depositAmount": 1
```
Comment (martinkersner) [inline contracts/migration/localhost/POR/20230831181923_deploy.json:8]: I think this parameter is long time deprecated.

File: contracts/migration/localhost/POR/20230831181923_deploy.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "deploy": {
+    "name": "e-Gold",
+    "timeout": 0,
+    "validator": "0x0000000000000000000000000000000000000000",
+    "decimals": 8,
```
Comment (martinkersner) [inline contracts/migration/localhost/POR/20230831181923_deploy.json:6]: @bayram98 Have you discussed this decimals number with Bryan or e-gold?

File: contracts/deploy/POR/POR.cjs [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  // DataFeedConsumerMock
+  await deploy(dataFeedConsumerMockName, {
+    contract: 'DataFeedConsumerMock',
+    args: [aggregatorProxyDeployment.address],
+    from: consumer,
+    log: true
+  })
+}
+
+func.id = 'deploy-aggregator'
```
Comment (martinkersner) [inline contracts/deploy/POR/POR.cjs:167]: I think the `id` and `tags` should be updated.

**Review by martinkersner — APPROVED** [review]
LGTM with minor comments!

File: contracts/migration/localhost/POR/20230831181923_deploy.json (hunk 2 above)
Comment (bayram98) [reply contracts/migration/localhost/POR/20230831181923_deploy.json:6]: Decimals not discussed it yet, I will put on agenda.
 
Current, deployment configuration is as a default settings. We will need to double check it later.

**Comment by bayram98** [comment]
@martinkersner Thank you for your comments.
Conflicts are resolved and I am going to merge the PR

**Pull request merged**